lib/websocket.go

```diff
@@ -36,8 +36,11 @@ func NewIRCWebSocket(wsUrl, origin string, tlsConfig *tls.Config) (IRCSocket, er
 
 	dialer := websocket.Dialer{
 		Subprotocols:    []string{"text.ircv3.net", "binary.ircv3.net"},
-		TLSClientConfig: tlsConfig,
 	}
+	if useTLS {
```

Comment:
this choice was made so that `ircdog.go` can unconditionally make a `tls.Config`, because both `--tls-noverify` asd `--tls-cert` want to put things in our `tls.Config`